### PR TITLE
Update Ember/CLI to v2.13.2 along with Babel 6

### PR DESCRIPTION
**Pull request by alexdiliberto**
@anandts88 This is the second PR to merge after you have already merged https://github.com/anandts88/ember-validator/pull/17 and published to NPM as release `v1.3.8`

Now merge this Ember 2.13.2 upgrade PR and release Ember Validator addon `v1.4.0`

Thank you!

/cc @jbailey4

### CODE REVIEW

**Review by jbailey4 — COMMENTED** [review]
@alexdiliberto just a couple suggestions

File: addon/mixins/numeric.js [hunk 1]
```diff
@@ -65,8 +65,8 @@ export default Mixin.create({
     }
 
     if (this.options.between && isArray(this.options.between)) {
-      first = this.options.range[0];
-      last = this.options.range[1];
+      let first = this.options.range[0];
+      let last = this.options.range[1];
```
Comment (jbailey4) [inline addon/mixins/numeric.js]: @alexdiliberto same as above, `let [ first, last ] = this.options.range;`

File: addon/mixins/numeric.js [hunk 2]
```diff
@@ -49,8 +49,8 @@ export default Mixin.create({
     }
 
     if (this.options.range && isArray(this.options.range)) {
-      first = this.options.range[0];
-      last = this.options.range[1];
+      let first = this.options.range[0];
+      let last = this.options.range[1];
```
Comment (jbailey4) [inline addon/mixins/numeric.js]: @alexdiliberto fyi, we could use array destructuring assignment here for conciseness:

`let [ first, last ] = this.options.range;`

**Comment by anandts88** [comment]
@jbailey4 Your comments are addressed.

**Pull request merged**